h2o-clustering/README.md

```diff
@@ -0,0 +1,35 @@
+# H2O Assisted clustering
+
+Assisted clustering is a special regime of H2O startup, invoked by setting the `H2O_ASSISTED_CLUSTERING_REST` environment
+variable to `True`. In this regime, `AssistedClusteringEmbeddedConfigProvider` is activated and spawns an HTTP endpoint at
+`http://h2o-node-host:<port:default 8080>/clustering/flatfile`, expecting a `POST` HTTP request with `Content-type: text/plain`
+and a standard H2O flatfile in the request body.
+
+### Example
+```
+curl --location --request POST 'localhost:8080/clustering/flatfile' \
+--header 'Content-Type: text/plain' \
+--data-raw '255.255.255.0:54321'
+```
+
+## Flatfile format
+
+This module itself does not make any assumptions about the flatfile formatting, besides the basic ones like
+an empty or missing request body. The flatfile is handed over to H2O and parsed by `NetworkInit.java` - which contains all the details.
+IPv4 and IPv6 formats are supported (not the shortened representation of IPv6). Both hostname/host ip and port in a format
+`hostname:port` must be present.
+
+### Example
+
+A flatfile with an IPv6 and IPv4 address with ports defined in both cases (mandatory).
+```
+[1200:0000:AB00:1234:0000:2552:7777:1313]:54321
+9.255.255.255:54321
+```
+
+## Distribution
```

Comment:
Hmm, I thought it would be better not to include it - that would give use the freedom to upgrade the code without upgrading H2O in deployments.

Reply:
A small change. And indeed, it is better. Will do.

Reply:
We can still include it in the k8s docker images, but I would leave it out of the jar to give us the option to swap them with less effort. WDYT?

Reply:
The Kubernetes Docker image could have it, yes. No harm done there. The Operator would have to know though (or it won't, as putting the same classes on CP twice does not harm anyone).

Reply:
As the operator has to do the check anyway, there is no need to do this now. I'd skip this optional step of adding this into the Kubernetes jar. The only user ATM is the operator. If someone goes as far as writing a custom clustering assistant, they can surely add one more `java -cp`